src/FamilyHubs.ServiceDirectory.Core/ServiceDirectory/Interfaces/IServiceDirectoryClient.cs

```diff
@@ -1,14 +1,17 @@
-﻿using FamilyHubs.ServiceDirectory.Core.ServiceDirectory.Models;
+﻿using System.Net;
+using FamilyHubs.ServiceDirectory.Core.ServiceDirectory.Models;
 using FamilyHubs.ServiceDirectory.Shared.Dto;
+using FamilyHubs.ServiceDirectory.Shared.Enums;
 using FamilyHubs.ServiceDirectory.Shared.Models;
-    
+using Microsoft.AspNetCore.Http;
+
 namespace FamilyHubs.ServiceDirectory.Core.ServiceDirectory.Interfaces;
 
 public interface IServiceDirectoryClient
 {
     // leaky, not clean, but this is our service, as opposed to a generic service that we might want to swap
     Task<PaginatedList<TaxonomyDto>> GetTaxonomies(CancellationToken cancellationToken = default);
-    Task<PaginatedList<ServiceDto>> GetServices(ServicesParams servicesParams, CancellationToken cancellationToken = default);
+    Task<(PaginatedList<ServiceDto> services, HttpResponseMessage? response)> GetServices(ServicesParams servicesParams, CancellationToken cancellationToken = default);
```

Comment:
Question: Just checking these changes are intentional? It's fine fi they are - I was expecting for the acceptance test project move only

Reply:
I'm not sure honestly. This PR is just a direct merge from the current release branch into the KPI release branch

Reply:
My mistake, wrong PR!